### PR TITLE
adding core-site files to DCOS version

### PR DESCRIPTION
**Pull request by abhay-agarwal**
These changes make it possible to run hue on your DCOS cluster (or any cluster for that matter)

### CODE REVIEW

File: conf/core-site.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
+  <property>
```
Comment (elingg) [inline conf/core-site.xml]: Does this property need to be added for Hue to work? If not, we should remove it for the reasons below.

I believe fs.default.name can be placed in hdfs-site.xml (the configuration can be ported there per @brndnmtthws) although we need to test it . This does need to be configurable based on the configured cluster name. Also, we need to update the readme to let the user know that hadoop fs -ls / will work as opposed to hadoop fs -ls hdfs://hdfs/.

**Comment by elingg** [comment]
Thanks @abhay-agarwal for contributing this! 

I left an important point above about the name of the hdfs cluster (making it configurable). We need to fix this before merging. Other than that, I think this will be the only way to configure HDFS to use Hue. I can't think of a better alternative as of now at least.

If we merge this for this release, we need to bump up the version number of HDFS (because I have code changes I have made recently). Then we need to build this and put it in a new bucket and update Universe and DCOS Image accordingly. Let's make sure we are very careful about integrating all these changes.

@adam-mesos, did you have any objections here or do you have a better alternative?

**Comment by elingg** [comment]
Spoke with some of the team, and we need to determine priorities and whether we need to integrate Hue now. Holding off until we get some more input.

File: conf/core-site.xml (hunk 1 above)
Comment (adam-mesos) [inline conf/core-site.xml]: +1, this is important for multi-hdfs installs, or custom names.

**Comment by adam-mesos** [comment]
I have no objection to merging this soon (after making the hdfs name configurable), since it's going to take a long time to get a proper solution for HDFS/Hue/Myriad to each provide their own custom Hadoop config changes. That solution may even require changes to Mesos itself.

Another alternative is that we push this change into the dcos-image if that's the only place we need it. But if other hdfs-mesos installs want to use Hue, this will be necessary in general.

**Comment by elingg** [comment]
Ok, the correct solution probably will take some time. I am concerned about adding yet another dependency or hack here, but if we make the hdfs name configurable, I think we can go with this temp solution. It is the best solution given our current situation. We do need to address my comment about making the hdfs name configurable.

**Comment by abhay-agarwal** [comment]
This isn't a dependency or hack, it's necessary to maintain the mesos/hdfs abstraction. In a typical scenario, your deployment manager would allow you to make changes to core-site (or an ops person would manually make the change). For DCOS, we need to include the best defaults for our users.

**Comment by elingg** [comment]
LGTM, Any final objections @adam-mesos? If not, I will merge!

**Comment by elingg** [comment]
@abhay-agarwal, the solution we have now is a hack because we are placing prepackaged configurations out of band of mesos on the slaves prior to installing the framework. We need a better overall solution for this, but as @adam-mesos mentioned, it will take a long time. That's why this is the best solution for now.

**Comment by adam-mesos** [comment]
> For DCOS, we need to include the best defaults for our users.

But this PR is a change for the hdfs framework. Would it be better suited for a DCOS repo, like DCOS-Image or pkgpanda?

**Comment by adam-mesos** [comment]
Ok, just saw that it was updated to only change the dcos-config template. Fine by me to merge. :ship:

**Comment by elingg** [comment]
Merging after the build passes :+1:

**Pull request merged**